charter.md

```diff
@@ -0,0 +1,34 @@
+#Charter for Working Group
+
+There are many situations in which it is desirable to share a digital credential with another person. For example, you may want to provide access to your vehicle to a Valet. You may also want to provide access to your home to your cat sitter. Or, you may want to share a hotel key with your spouse. Today, no such standardized method exists in a cross-platform, multidisciplinary capacity. 
+
+The WG charter includes the definition and standardization of a protocol that will facilitate such credential transfers from individual to individual. The protocol will leverage a “relay server” to transfer data from sender to recipient. The scope of the transfer is limited to a single origin device and a single destination device.
+
+Privacy goals include:
+
+- The relay server should not see sensitive details of the share
+- The relay server should not be able to provision the credential itself, acting as an intermediary for the recipient (MiTM)
+- The relay server should not know the identity of the sender nor receiver
+- Device OEMs should not be able to identity the social graph of whom has shared with whom
```

Comment:
I don't think this is needed. If the point above it is upheld then the relay server wouldn't be able to build a social graph, and our internal privacy concerns and mitigations seen unrelated to the EITF.

Reply:
+1

Reply:
Agreed